templates/repo/commit_page.tmpl

```diff
@@ -201,7 +214,12 @@
 								<span class="ui text gt-mr-3">{{.locale.Tr "repo.commits.signed_by_untrusted_user_unmatched"}}:</span>
 							{{end}}
 							{{avatar $.Context .Verification.SigningUser 28 "gt-mr-3"}}
-							<a href="{{.Verification.SigningUser.HomeLink}}"><strong>{{.Verification.SigningUser.GetDisplayName}}</strong></a>
+							{{if .Verification.SigningUser.FullName}}
+								{{/* Use the same scheme as the one used in commits. */}}
+								<a href="{{.Verification.SigningUser.HomeLink}}"><strong> {{.Verification.SigningUser.FullName}} ({{.Verification.SigningUser.GetDisplayName}})</strong></a>
```

Comment:
I guess it could be like this (or `Fullname` (`Name`) ) ?

```suggestion
								<a href="{{.Verification.SigningUser.HomeLink}}"><strong> {{.Verification.SigningUser.Name}} ({{.Verification.SigningUser.FullName}})</strong></a>
```


Otherwise, GetDisplayName sometimes returns "Name", sometimes returns "FullName".

Overall LGTM while there is still a question in my mind: when to use `user.Name` , when to use `user.FullName` ? (see another comment)

Reply:
IIRC it works like this:

```go
if DefaultShowFullName && FullName != "" {
    return FullName
} else {
    return Name
}
```

Reply:
I didn't mean following `DefaultShowFullName` is a must. As long as the new display logic is complete, consistent and reasonable, then it's good 😄